packages/env/lib/cli.js

```diff
@@ -60,26 +65,32 @@ const withSpinner =
 				) {
 					// Error is a docker compose error. That means something docker-related failed.
 					// https://github.com/PDMLab/docker-compose/blob/HEAD/src/index.ts
-					spinner.fail(
-						'Error while running docker compose command.'
-					);
-					if ( error.out ) {
-						process.stdout.write( error.out );
-					}
-					if ( error.err ) {
-						process.stderr.write( error.err );
+					if ( ! silent ) {
+						spinner.fail(
```

Comment:
This isn't good because it suppresses errors when Docker isn't running. Perhaps only the first condition should be suppresses.

Reply:
Ah, so should this not be suppressed if `--quiet`?